rs/rosetta-api/runner/src/lib.rs

```diff
@@ -22,9 +20,18 @@ impl RosettaContext {
         self.port
     }
 
-    pub fn kill(self) {
+    /// Kills the rosetta process and deletes the state directory.
+    /// All traces of the rosetta process will be removed.
+    pub fn kill_rosetta_context(self) {
         drop(self._proc);
     }
+
+    /// Kills the process in which the rosetta server and blocks synchronizer are running.
+    /// Leaves the state directory untouched.
+    /// This is useful when you want to restart the rosetta server with the same state directory. (Load existing blocks from storage)
+    pub fn kill_rosetta_process(&mut self) {
+        self._proc.0.kill().expect("Failed to kill rosetta process");
```

Comment:
Since `_proc` is now (actively) read, could the underscore be removed from the variable name (i.e., just call it `proc`)?

Reply:
Yes